site/_posts/2019-09-20-Velero-v1-1-on-PKS.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Velero v1.1 backing up and restoring apps on Pivotal Container Service (PKS)
+image: /img/posts/PKS-logo.png
+excerpt: Learn how Velero can be used on PKS, the Pivotal Container Service which deploys and managages Kubernetes clusters on vSphere.
+author_name: Cormac Hogan
+author_avatar: /img/contributors/cormac-pic.png
+categories: ['kubernetes']
+# Tag should match author to drive author pages
+tags: ['Velero', 'Cormac Hogan', 'how-to']
+---
+Velero version 1.1 provides support to backup Kubernetes applications orchestrated on top of PKS, the Pivotal Container Service for vSphere. This post will provide detailed information on how to install and configure Velero to backup and restore a stateless application (`nginx`) that is running in a Kubernetes cluster deployed by PKS on vSphere. At this time there is no vSphere plugin for snapshotting stateful applications during a Velero backup. In this case, we rely on a third party program called `restic`. However this post does not include an example of how to backup a stateful application. That is available in another tutorial which can be found [here](./2019-20-09-Velero-v1-1-Stateful-Backup-vSphere.md).
```

Comment:
The `here` link at the end doesn't seem to be working in the site preview - we probably want to make it an absolute  URL, e.g. `velero.io/velero-v1-1-stateful-backup-vsphere/`

Reply:
Fixed